Add ansible.cfg

alanorth opened the pull request:
Pipelining makes ansible run tasks much faster[0], and as of Ansible 2.0 or 2.1 or so, it now works on older hosts that have `requiretty` in their sudo config[1]. Also, disable the creation of those stupid retry files.

[0] https://docs.ansible.com/ansible/intro_configuration.html#pipelining
[1] https://github.com/ansible/ansible/pull/13200

alanorth committed:
Add ansible.cfg

Pipelining makes ansible run tasks much faster, and as of Ansible
2.0 or 2.1 or so, it now works on older hosts that have requiretty
in their sudo config[0]. Also, disable the creation of those stupid
retry files.

[0] https://github.com/ansible/ansible/pull/13200

oguya commented:
I actually have one with `retry_files_enabled=False` to prevent ansible from creating those annoying $playbook.retry files.

Commit:
ansible.cfg: Add inventory parameter

Instead of relying on ANSIBLE_HOSTS env. var. to provide the path of
ansible inventory file, we should use `inventory' parameter.

Signed-off-by: James Oguya <j.oguya@cgiar.org>

Commit:
misc_plays/: Add symlink to ansible.cfg

We also have a couple of 'one-off' playbooks that would also make good
use of ansible.cfg config. file.

Signed-off-by: James Oguya <j.oguya@cgiar.org>

alanorth commented:
@oguya Would you like a cookie?

oguya commented:
@alanorth what's up with the 'cookie'?

alanorth commented:
I was teasing you for flossing in your comment. 😉
